components/FormikFields/PaperInputPicker/index.js

```diff
@@ -34,6 +37,65 @@ const PaperInputPicker = ({
 
   const [location, setLocation] = React.useState({ latitude: 0, longitude: 0, altitude: 0 });
   const [locationLoading, setLocationLoading] = React.useState(false);
+  const [numberedQestions, setNumberedQuestions] = React.useState({})
+  const [loopPosition, setLoopPosition] = React.useState(-1);
+  const [questionsToRepeat, setQuestionsToRepeat] = React.useState([]);
+  const [loopsAdded, setLoopsAdded] = React.useState(0);
+  const [additionalQuestions, setAdditionalQuestions] = useState([]);
+
+  useEffect(() => {
+    let fieldsNumberedJson = {}
+    // console.log(config.fields)
+    config.fields.forEach((field, index) => {
+      fieldsNumberedJson[index] = field;
+    })
+    setNumberedQuestions(fieldsNumberedJson)
```

Comment:
turn into function

Reply:
Should be taken care of. Lmk if furhter changes required